README.adoc

```diff
@@ -15,30 +16,38 @@ https://coveralls.io/github/CS2103-AY1819S1-T09-2/main?branch=master[image:https
 https://www.codacy.com/app/leeyjjoel/main?utm_source=github.com&amp;utm_medium=referral&amp;utm_content=CS2103-AY1819S1-T09-2/main&amp;utm_campaign=Badge_Grade[image:https://api.codacy.com/project/badge/Grade/1bb58958c22c473dbf75c560d5b71612[Codacy Badge]]
 https://gitter.im/se-edu/Lobby[image:https://badges.gitter.im/se-edu/Lobby.svg[Gitter chat]]
 
+
 Anakin is a spaced repetition flashcard program which makes remembering things easy.
+Here's what some of our users have to say:
+
+[quote, Tan Xiao Ming, Computer Science Graduate]
+Amazing work. Simply amazing.
+
+[quote, Donnie T., Political Science Student]
+The greatest CLI app, maybe, ever!
+
+[quote, A.Aron Tan, Lecturer]
+They did it.
```

Comment:
Perhaps we could have a quote by Tony Stark commenting "Better than Pepper Potts!"

Reply:
Change has been implemented